pkg/expect/expect_test.go

```diff
@@ -72,11 +85,13 @@ func TestExpectFuncTimeout(t *testing.T) {
 	}
 
 	err = ep.Close()
-	require.ErrorContains(t, err, "unexpected exit code [143] after running [/usr/bin/tail -f /dev/null]")
+	require.ErrorContains(t, err, fmt.Sprintf("unexpected exit code [143] after running [%s -f /dev/null]", tailpath))
```

Comment:
Could we just remove absolute paths?
```suggestion
	require.ErrorContains(t, err, "unexpected exit code [143] after running")
```

Reply:
It does make it simpler not printing verbose error. Changed it here and elsewhere, thanks.